docs/development/local_setup.md

```diff
@@ -306,12 +306,6 @@ apiservice.apiregistration.k8s.io/v1alpha1.seedmanagement.gardener.cloud created
 apiservice.apiregistration.k8s.io/v1alpha1.settings.gardener.cloud created
 ```
 
-Optionally, you can switch off the `Logging` feature gate of Gardenlet to save resources:
-
-```bash
-sed -i -e 's/Logging: true/Logging: false/g' dev/20-componentconfig-gardenlet.yaml
-```
-
```

Comment:
Instead of removing this, what about mentioning the new `logging.enabled` option in the config?

Reply:
Addressed in 8c9ee3ad536361a54bdf56825cd1cb78a2bc8b72